rdrf/rdrf/settings.py

```diff
@@ -24,6 +24,7 @@
 
 # General site config
 PRODUCTION = env.get("production", False)
+STAGING = env.get("staging", False)
```

Comment:
I would maybe make this setting a bit more generic like `SHOW_BANNER` or `SHOW_NONPROD_BANNER`. We might decide to use the banner in other environments, for example.
Thoughts, @ScriptSmith ?

Reply:
Yep, `SHOW_BANNER` works better :+1: